src/lib/services/charge-versions.js

```diff
@@ -104,6 +105,11 @@ const persist = async chargeVersion => {
   )
   persistedChargeVersion.chargeElements = await Promise.all(tasks)
 
+  console.log('Charge Version', persistedChargeVersion)
+  console.log('AHHHH Im in service!!!!')
```

Comment:
Need to remove the `console.log()`s.

Reply:
I didn't realise this was up for review sorry, not finished with it! <img src="https://media1.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif"/>